images/win/scripts/Installers/Install-Msys2.ps1

```diff
@@ -0,0 +1,80 @@
+################################################################################
+##  File:  Install-Msys2.ps1
+##  Desc:  Install Msys2 and 64-bit gcc, cmake, & llvm (32-bit commented out)
+################################################################################
+
+# References
+# https://github.com/msys2/MINGW-packages/blob/master/azure-pipelines.yml
+# https://packages.msys2.org/group/
+
+$env:origPath = $env:PATH
+$env:gitPath  = "C:\Program Files\Git"
+
+# get info from https://sourceforge.net/projects/msys2/files/Base/x86_64/
+$msys2Uri  = "http://repo.msys2.org/distrib/x86_64/msys2-base-x86_64-20190524.tar.xz"
+$msys2File = "C:\Windows\Temp\msys2.tar.xz"
+
+# Download the latest msys2 x86_64
+Write-Host "Starting msys2 download"
+(New-Object System.Net.WebClient).DownloadFile($msys2Uri, $msys2File)
+Write-Host "Finished download"
+
+$msys2FileU = "/$msys2File".replace(':', '')
```

Comment:
Why we need this line?
`$msys2File = "C:\Windows\Temp\msys2.tar.xz"` I don't see any `:`

Reply:
C:\ => C\